tests/functional/u/unbalanced_tuple_unpacking.py

```diff
@@ -106,3 +107,12 @@ def test_issue_559():
     from ctypes import c_int
     root_x, root_y, win_x, win_y = [c_int()] * 4
     return root_x, root_y, win_x, win_y
+
+
+class MyClass(NamedTuple):
+    first: float
+    second: float
+
+    def sum_3d(self):
+        first, second, third = self # [unbalanced-tuple-unpacking]
+        return first + second + third
```

Comment:
Could you add two more functions here?
* One that doesn't emit a warning. (Basically the one from the other test file.)
* Unbalanced-unpacking but the other way, ie. trying to unpack 3 into 2 variables.